pkg/apis/pingcap/v1alpha1/types.go

```diff
@@ -1138,6 +1138,8 @@ type BackupSpec struct {
 	UseKMS bool `json:"useKMS,omitempty"`
 	// Specify service account of backup
 	ServiceAccount string `json:"serviceAccount,omitempty"`
+	// CleanData denotes whether to clean backup data before the object is deleted from the cluster
```

Comment:
Please help update the examples in `manifests/backup/`.

Reply:
addressed in https://github.com/pingcap/tidb-operator/pull/2892/commits/ee29d324b6fc2e46c4587c2bb4ecd9a83947ad4d